app/models/user.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define(
+    'user',
+    {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: DataTypes.INTEGER
+      },
+      firstName: { alowNull: false, type: DataTypes.STRING },
+      lastName: { alowNull: false, type: DataTypes.STRING },
+      email: { alowNull: false, type: DataTypes.STRING, unique: true },
+      password: { alowNull: false, type: DataTypes.STRING }
+    },
+    {}
```

Comment:
Add `paranoid:true`. Check this in documentation for models: http://docs.sequelizejs.com/manual/tutorial/models-definition.html

Reply:
👍  We don't want to really delete this object.